### DISCUSSION TITLE
ThemeEditor: Full icon replacement

### DISCUSSION THREAD

**Discussion by WolfgangDpunkt** [Suplanus/Cuterdio#127] [Ideas]
Thanks for your quick response in the iOS Appstore!
In fact, I already have a cover of my choice in the theme editor. This is also displayed.
When I start the app, the unicorn will be displayed in full screen for a short time and only then it will change to my desired cover. Since I start the app frequently, this quick change of images is a bit irritating, so it would be nice if you could hide the unicorn completely, as nice as it is.  
As an alternative, more discreet icon I would recommend an icon from the collection of the Noun-Project, the icons there are also free for commercial use, if you name them.
Personally I find the vintage radios quite chic: https://thenounproject.com/term/retro-radio

---
_Johann Weiher: I have translated your german text_
>Original:
Danke für deine schnelle  Reaktion im iOS-Appstore!
Ich habe tatsächlich bereits ein Cover meiner Wahl im Theme-Editor hinterlegt. Das wird auch angezeigt.
Wenn ich die App starte blendet sich trotzdem kurz das Einhorn bildschirm-füllend ein und erst dann wechselt es zu meinem Wunschcover. Da ich die App häufig starte, ist dieser schnelle Wechsel der Bilder etwas irritierend, daher wäre es schön, wenn man das Unicorn komplett verstecken könnte, so sympathisch es auch ist.  
Als alternatives, dezenteres Icon würde ich ein Symbol aus der Sammlung des Noun-Project empfehlen, die Icons dort sind auch für die kommerzielle Verwendung frei verwendbar, bei Namensnennung.
Die Vintage-Radios finde ich persönlich recht schick: https://thenounproject.com/term/retro-radio/

**Comment by Suplanus** [comment]
Der Startup-Screen ist leider nicht so einfach zu manipulieren. Hätte das gerne schon gemacht...
Ich schau mal was ich machen kann 🦄

Icon von TheNounProject ist eine gute Idee. Du kannst aber das Icon jetzt schon selbst erstellen. Siehe Workaround hier: #27

**Comment by WolfgangDpunkt** [comment]
Ggf. kann man den Startup-Screen ja als default etwas neutraler gestalten und erst nach Start kommt das Unicorn (es sei denn man tauscht es in den Einstellungen).
Cool, Danke für den Hinweis auf den Workaround!
